pages/content/amp-dev/documentation/guides-and-tutorials/start/bento_guide.md

```diff
@@ -64,27 +60,33 @@ Each Bento component has a small CSS library you must include to guarantee prope
 <link rel="stylesheet" type="text/css" href="https://cdn.ampproject.org/v0/amp-bento-component-name-1.0.css">
 <style>
  custom styles
-</style> 
+</style>
 ```
 
+Tip: For the best rendering performance inline the Bento component CSS directly into the page to avoid a render-blocking request to `cdn.ampproject.org` on page load.
```

Comment:
Should this be `[tip type="note|important"]` ?

Reply:
This is the super secret short code notation for tips. I've changed this to the standard format though. Old habits...